package.json

```diff
@@ -30,68 +30,61 @@
+    "compression": "^1.7.4",
+    "enzyme": "^3.11.0",
+    "eslint-config-jest-enzyme": "^7.1.2",
+    "eslint-plugin-flowtype": "^8.0.3",
+    "eslint-plugin-jest": "^26.8.7",
+    "eslint-plugin-jsx-a11y": "^6.6.1",
+    "eslint-plugin-react": "^7.30.1",
+    "eslint-plugin-react-hooks": "^4.6.0",
     "express-watcher": "^1.0.3",
-    "http-proxy-middleware": "^1.0.6",
-    "husky": "^4.3.0",
-    "jest-axe": "^4.0.0",
-    "jest-environment-enzyme": "^7.0.2",
-    "jest-enzyme": "^7.0.2",
-    "ndb": "^1.0.44",
-    "node-sass": "^6.0.0",
-    "nodemon": "^2.0.1",
-    "npm-run-all": "^4.0.2",
-    "postcss": "^8.0.9",
-    "postcss-cli": "^8.1.0",
-    "react-scripts": "4.0.3",
-    "react-test-renderer": "^17.0.1",
-    "sass-lint": "^1.12.1",
-    "script-ext-html-webpack-plugin": "^2.1.3",
-    "source-map-explorer": "^2.5.0"
+    "http-proxy-middleware": "^2.0.6",
+    "husky": "^8.0.1",
+    "jest-axe": "^6.0.0",
+    "jest-environment-enzyme": "^7.1.2",
+    "jest-enzyme": "^7.1.2",
+    "ndb": "^1.1.5",
+    "nodemon": "^2.0.19",
+    "npm-run-all": "^4.1.5",
+    "postcss": "^8.4.16",
+    "postcss-cli": "^10.0.0",
+    "react-scripts": "5.0.0",
+    "react-test-renderer": "^18.2.0",
+    "sass": "^1.54.6",
+    "sass-lint": "^1.13.1",
+    "script-ext-html-webpack-plugin": "^2.1.5",
+    "source-map-explorer": "^2.5.2"
   },
   "dependencies": {
-    "@babel/cli": "^7.11.6",
-    "@babel/core": "^7.11.6",
-    "@babel/parser": "^7.11.5",
-    "@babel/plugin-proposal-class-properties": "^7.4.4",
-    "@babel/plugin-proposal-export-namespace-from": "^7.2.0",
-    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.10.1",
-    "@babel/plugin-proposal-optional-chaining": "^7.11.0",
-    "@babel/plugin-proposal-throw-expressions": "^7.2.0",
-    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
-    "@babel/polyfill": "^7.11.5",
-    "@babel/preset-env": "^7.11.5",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/register": "^7.11.5",
-    "@babel/types": "^7.11.5",
-    "@marvelapp/react-ab-test": "^2.3.0",
-    "@testing-library/jest-dom": "^5.11.4",
+    "@babel/cli": "^7.18.10",
+    "@babel/core": "^7.18.13",
+    "@babel/parser": "^7.18.13",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-proposal-export-namespace-from": "^7.18.9",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.18.6",
+    "@babel/plugin-proposal-optional-chaining": "^7.18.9",
+    "@babel/plugin-proposal-throw-expressions": "^7.18.6",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/preset-env": "^7.18.10",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/register": "^7.18.9",
+    "@babel/types": "^7.18.13",
+    "@babel/eslint-parser": "^7.18.9",
+    "babel-plugin-dynamic-import-node": "^2.3.3",
+    "babel-plugin-lodash": "^3.3.4",
+    "babel-plugin-module-resolver": "^4.1.0",
+    "babel-preset-react-app": "^10.0.1",
+    "@marvelapp/react-ab-test": "^3.1.0",
```

Comment:
I was upgrade to version 3 that support hooks and react 17. Maybe you will want make changes

https://github.com/marvelapp/react-ab-test/releases/tag/v3.0.0